tests/reg_tests/reg_test_classes.py

```diff
@@ -31,5 +41,6 @@ def train(self):
         for test in tests:
             test_func = getattr(self, test)
             test_func()
+            self.handler.add_metadata(self.CFDSolver.options)
```

Comment:
I agree with Anil. There should be no manual changes. I think we should just pop the relevant options before adding them. (Also, use `getOptions` instead of directly accessing the options dictionary.)